src/components/WalletDropdown/MiniPortfolio/Activity/parseLocal.tsx

```diff
@@ -79,7 +81,9 @@ function parseApproval(
 ): Partial<Activity> {
   // TODO: Add 'amount' approved to ApproveTransactionInfo so we can distinguish between revoke and approve
   const currency = getCurrency(approval.tokenAddress, chainId, tokens)
-  const descriptor = t`${currency.symbol ?? currency.name}`
+  const symbol = currency?.symbol
+  const name = currency?.name ?? 'Unknown'
+  const descriptor = t`${symbol ?? name}`
```

Comment:
Only 'Unknown' should be translated. You could also make this inline, which I think is more readable

```suggestion
  const descriptor = currency?.symbol ?? currency?.name ?? t`Unknown`
```